src/ast/passes/codegen_llvm.cpp

```diff
@@ -1706,7 +1721,13 @@ void CodegenLLVM::binop_int(Binop &binop)
     default:
       LOG(BUG) << "\"" << opstr(binop) << "\" was handled earlier";
   }
-  // Using signed extension will result in -1 which will likely confuse users
+
+  return;
```

Comment:
Also seems like this early return changes the expression for other operations. Is that intentional?